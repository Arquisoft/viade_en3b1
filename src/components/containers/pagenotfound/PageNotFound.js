import React from 'react';
import "typeface-roboto";
import { Link, Typography } from '@material-ui/core';

const PageNotFound = () => {
    
    return (
        <div style={{marginLeft:'5vh', marginTop:'5vh'}}>
            <Typography variant="h4">Sorry, there's nothing to see here</Typography>
            <p>Go back to <Link href={"#/"}>Home</Link></p>
        </div>
    );
};

export default PageNotFound;
