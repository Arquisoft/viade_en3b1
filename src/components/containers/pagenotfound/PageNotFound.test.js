import PageNotFound from "./PageNotFound";
import { Typography, Link } from "@material-ui/core";

test("PageNotFound", () => {
    expect(PageNotFound).toBeDefined();
});

test("Renders Correct", () => {
    const wrapper = render(<PageNotFound />);
    expect(wrapper).toMatchSnapshot();
});

test("Home Link", () => {
    const wrapper = mount(<PageNotFound />);
    const element = wrapper.find(Link);
    
    expect(element.exists()).toBeTruthy();
    expect(element.text()).toEqual("Home");
});