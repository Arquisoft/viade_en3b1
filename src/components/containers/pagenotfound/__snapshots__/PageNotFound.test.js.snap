// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Correct 1`] = `
<div
  style="margin-left:5vh;margin-top:5vh;text-align:center"
>
  <img
    alt="404 logo"
    src="404.svg"
  />
  <div
    class="MuiBox-root MuiBox-root-2"
  >
    Go back to 
    <a
      class="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      href="#/"
      style="color:#27DEBF"
    >
      Home
    </a>
  </div>
</div>
`;
